scripts/util/k8s.py

```diff
@@ -80,3 +80,35 @@ def get_latest_releases_by_minor() -> Dict[str, str]:
             latest_by_minor[key] = tag
 
     return latest_by_minor
+
+
+def get_all_releases_after(release) -> set[str]:
+    """
+    Get all releases after the input release.
+    If the input release is invalid, the output will be empty.
+    """
+    releases: set[str] = set()
+    version_regex = re.compile(r"^v?(\d+)\.(\d+)\..+")
+
+    match = re.match(r"^(\d+)\.(\d+)$", release)
+    if not match:
+        return releases
+    least_major, least_minor = match.groups()
```

Comment:
Let's just not go down the regex road for parsing what are version strings

```suggestion
    try:
        version = Version(release)
    except packaging.version.InvalidVersion:
        raise ValueError(f"{release} is not a valid version")
```

Reply:
Done